js/ui/pivot_grid/ui.pivot_grid.js

```diff
@@ -1349,14 +1360,15 @@ const PivotGrid = Widget.inherit({
                     }
                 }
 
+                const scrollingOptions = that.option('scrolling');
                 if(scrollingOptions.mode === 'virtual') {
                     that._setVirtualContentParams(scrollingOptions, resultWidths, resultHeights, groupWidth, groupHeight, that._hasHeight, rowsAreaWidth);
                 }
 
                 const updateScrollableResults = [];
                 that._dataArea.updateScrollableOptions({
-                    useNative: !!scrollBarInfo.scrollBarUseNative,
-                    useSimulatedScrollbar: !scrollBarInfo.scrollBarUseNative,
+                    useNative: scrollBarUseNative,
+                    useSimulatedScrollbar: !scrollBarUseNative,
```

Comment:
Can we pass 'useNative' only?
(updateScrollableOptions will manually add 'useSimulatedScrollbar' to its 'restOptions')

Reply:
I may set 'false' value to 'useSimulatedScrollbar' option directly on first initialization. It needs to override calculated default value inside scrollable if we didn't set custom value.

Current behavior:
- useSimulatedScrollbar don't use in **simulated** strategy.
- we use useSimulatedScrollbar: !useNative setting to set false value for native strategy only